### PR TITLE
setup new mw* with stunnel & prometheus

### PR DESCRIPTION
**Pull request by RhinosF1**
Note: add to modules/varnish/files/stunnel/stunnel.conf when IPs exist

### CODE REVIEW

**Comment by paladox** [comment]
Note that the old infra and new infra won't interact with each other. So the new infra won't connect to the old puppet server so the old infra won't have the data to generate known host filees using the new vms key/or won't know about them at all.

**Comment by RhinosF1** [comment]
> Note that the old infra and new infra won't interact with each other. So the new infra won't connect to the old puppet server so the old infra won't have the data to generate known host filees using the new vms key/or won't know about them at all.

I assume we'll have an icinga-new like last time and we can just downtime the checks from the wrong DC

File: modules/base/templates/icinga/nrpe.cfg.erb [hunk 1]
```diff
@@ -56,6 +56,14 @@ command[check_stunnel_mw12]=/usr/lib/nagios/plugins/check_http -H localhost:8092
...
 command[check_stunnel_mwtask1]=/usr/lib/nagios/plugins/check_http -H localhost:8089
 command[check_stunnel_test3]=/usr/lib/nagios/plugins/check_http -H localhost:8091
+command[check_stunnel_mw101]=/usr/lib/nagios/plugins/check_http -H localhost:8094
+command[check_stunnel_mw102]=/usr/lib/nagios/plugins/check_http -H localhost:8095
+command[check_stunnel_mw111]=/usr/lib/nagios/plugins/check_http -H localhost:8096
+command[check_stunnel_mw112]=/usr/lib/nagios/plugins/check_http -H localhost:8097
+command[check_stunnel_mw121]=/usr/lib/nagios/plugins/check_http -H localhost:8098
+command[check_stunnel_mw122]=/usr/lib/nagios/plugins/check_http -H localhost:8099
+command[check_stunnel_mwtask101]=/usr/lib/nagios/plugins/check_http -H localhost:8100
+command[check_stunnel_test111]=/usr/lib/nagios/plugins/check_http -H localhost:8101
```
Comment (Universal-Omega) [inline modules/base/templates/icinga/nrpe.cfg.erb:66]: test101

File: modules/base/templates/icinga/nrpe.cfg.erb [hunk 2]
```diff
@@ -56,6 +56,14 @@ command[check_stunnel_mw12]=/usr/lib/nagios/plugins/check_http -H localhost:8092
 command[check_stunnel_mw13]=/usr/lib/nagios/plugins/check_http -H localhost:8093
 command[check_stunnel_mwtask1]=/usr/lib/nagios/plugins/check_http -H localhost:8089
 command[check_stunnel_test3]=/usr/lib/nagios/plugins/check_http -H localhost:8091
+command[check_stunnel_mw101]=/usr/lib/nagios/plugins/check_http -H localhost:8094
+command[check_stunnel_mw102]=/usr/lib/nagios/plugins/check_http -H localhost:8095
+command[check_stunnel_mw111]=/usr/lib/nagios/plugins/check_http -H localhost:8096
+command[check_stunnel_mw112]=/usr/lib/nagios/plugins/check_http -H localhost:8097
+command[check_stunnel_mw121]=/usr/lib/nagios/plugins/check_http -H localhost:8098
+command[check_stunnel_mw122]=/usr/lib/nagios/plugins/check_http -H localhost:8099
+command[check_stunnel_mwtask101]=/usr/lib/nagios/plugins/check_http -H localhost:8100
```
Comment (Universal-Omega) [inline modules/base/templates/icinga/nrpe.cfg.erb:65]: mwtask111

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Pull request merged**